cloud/new_ec2_integration_test.go

```diff
@@ -12,22 +12,52 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestSpotInstanceIntegration(t *testing.T) {
+func fetchTestDistro() *distro.Distro {
+	return &distro.Distro{
+		Id:       "test_distro",
+		Arch:     "linux_amd64",
+		WorkDir:  "/data/mci",
+		PoolSize: 10,
+		Provider: evergreen.ProviderNameEc2Spot,
+		ProviderSettings: &map[string]interface{}{
+			"ami":            "ami-c7e7f2d0",
+			"instance_type":  "t1.micro",
+			"key_name":       "mci",
+			"bid_price":      .005,
+			"security_group": "default",
+		},
+
+		SetupAsSudo: true,
+		Setup:       "",
+		Teardown:    "",
+		User:        "root",
+		SSHKey:      "",
+	}
+}
+
+// This spawns (and terminates) an actual on-demand instance in AWS. Since AWS bills for a minimum
+// of 1 minute, this will cost approximately $.0002 each time it is run (t1.micro
+// $0.0116 per hour / 60 minutes). Since the price is low, we run this each time
+// the integration cloud tests are run.
```

Comment:
is there any rounding that might impact things? 

where are the credentials pulled from.

Reply:
> is there any rounding that might impact things?

I don't think so.

> where are the credentials pulled from.

From the settings file, like any other integration test.